SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -66,6 +68,8 @@
 
   @Inject
   public SingularityValidator(SingularityConfiguration configuration, DeployHistoryHelper deployHistoryHelper, RequestManager requestManager) {
+    this.configuration = configuration;
```

Comment:
the variable is already in SingularityConfiguration, it's accessed via `configuration.getDefaultBounceExpirationMinutes()`